README.md

```diff
@@ -2,42 +2,41 @@
 ![Build Status](https://img.shields.io/circleci/project/github/italia/developers-italia-onboarding/master.svg?style=flat
 ) ![Issues](https://img.shields.io/github/issues/italia/developers-italia-onboarding.svg) ![License](https://img.shields.io/github/license/italia/developers-italia-onboarding.svg?style=flat)
 
-Tramite l'applicazione di onboarding, le PA possono registrare i propri repository di code hosting sul portale [Developers Italia](https://innovazione.gov.it/it/progetti/developers-italia/). In questo modo è possibile aggiungere il repository alla lista indicizzata dal *crawler* del portale, che popolerà il catalogo del riuso. 
+The onboarding tool allows Italian Public Administrations to register their own code repositories in [Developers Italia](https://innovazione.gov.it/it/progetti/developers-italia/). As such, repositories will be scanned by the [crawler](https://github.com/italia/developers-italia-backend) that will feed the reuse catalog.
 
 ## Developers Italia
 
-Ulteriori informazioni relative al progetto Developers Italia possono essere trovate sul sito del [https://innovazione.gov.it](https://innovazione.gov.it/it/progetti/developers-italia/).
+More informations about Developers Italia can be found on the [website https://innovazione.gov.it](https://innovazione.gov.it/it/progetti/developers-italia/).
 
-## Flow applicativo
+## Application flow
 
-* Inserimento da parte della PA dei dati relativi all'organizzazione. Utilizzando la funzionalità di *ricerca amministrazione* i dati sono
-automaticamente inseriti nei campi *Codice iPA, Amministrazione e PEC*. 
+* A Public Administration inserts its data into the system. Leveraging the functionality, the fields *ricerca amministrazione*, *Codice iPA*, *Amministrazione* and *PEC* are automatically mapped
 
-* Dopo aver selezionato il pulsante *Registra*, l'applicativo invia una PEC all'indirizzo email dell'amministrazione indicato nel form. 
+* After clicking *Registra*, the application sends a *PEC* to the email address that the Public Administration has specified in the form
```

Comment:
The email address is provided from IndicePA, a database feeded by each PA itself, and filled into the form by just selecting the right PA from *ricerca amministrazione* field.